src/NCronJob/Configuration/Builder/JobOption.cs

```diff
@@ -35,3 +35,11 @@ internal sealed class JobOption
     /// </summary>
     public string? Name { get; set; }
 }
+
+#pragma warning disable S2094 // Classes should not be empty
+
+// TODO: To be dropped in v5.
+// Workaround for backward compatibility purposes
+// see https://github.com/NCronJob-Dev/NCronJob/pull/249
+internal class JobOptionParameter : JobOption { }
```

Comment:
Why is this needed? It is just a marker right now?

Reply:
A marker indeed.

It's easiest way I've found to support the requirement from https://github.com/NCronJob-Dev/NCronJob/pull/249#discussion_r1997542405

Without it, the newly added test fail.

![image](https://github.com/user-attachments/assets/82e26474-690e-4fac-a9a5-862b13c88eae)

If you can come up with an alternative implementation, I'm a taker.

Reply:
Okay - while this may seem weird, but then, I am fine with not overriding `Parameter` given that we didn't need this in the first place (see: https://github.com/NCronJob-Dev/NCronJob/pull/249#discussion_r1997542405)